code/modules/reagents/Chemistry-Reagents/Consumable-Reagents/Drink-Reagents/Alcohols.dm

```diff
@@ -19,24 +19,14 @@ datum/reagent/consumable/ethanol
 	description = "A well-known alcohol with a variety of applications."
 	color = "#404030" // rgb: 64, 64, 48
 	nutriment_factor = 0
-	var/boozepwr = 10 //lower numbers mean the booze will have an effect faster.
+	var/boozepwr = 4 //lower numbers mean the booze will have an effect faster.
```

Comment:
Should probably change that comment, since boozepwr now works opposite of what it meant.